e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -55,6 +57,23 @@ describe("banner", () => {
       });
   });
 
+  // Until we enable multi-browser support, this repro will be skipped by Cypress in CI
+  // Issue was specific to Firefox only - it is still possible to test it locally
+  it(
+    "should show info sidebar correctly on Firefox",
+    { browser: "firefox" },
+    function () {
+      visitDashboard(1);
+      cy.findByRole("main").findByText("Loading...").should("not.exist");
+      cy.findByRole("main").icon("info").click();
+
+      rightSidebar().within(() => {
```

Comment:
This asserts that the info sidebar is visible 
![image](https://github.com/metabase/metabase/assets/1937582/35d6df69-ad1b-4d55-ba1b-39514c837966)